x-pack/legacy/plugins/apm/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
do we need this file right now?

Reply:
No but it is auto-generated by cypress on `yarn install` afair